cli/pcluster/commands.py

```diff
@@ -211,6 +146,10 @@ def create(args):  # noqa: C901 FIXME!!!
         if bucket_name:
             cfn_params["ResourcesS3Bucket"] = bucket_name
 
+        # TODO: to refactor
+        if cluster_section.get_param_value("scheduler") == "slurm":
+            _upload_hit_resources(bucket_name, pcluster_config, storage_data.s3_params)
```

Comment:
this call is already in _create_bucket_with_resources

Reply:
Fixed